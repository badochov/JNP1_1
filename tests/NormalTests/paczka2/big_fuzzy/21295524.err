Error in line 17: W7n9 A822 95,8
Error in line 29: W7n9 A822 21,5
Error in line 26: k3o A20 87,7
Error in line 39: W7n9 A822 70,4
Error in line 41: a3c A20 63,6
Error in line 32: PqM A822 65,0
Error in line 54: W7n9 S5H5 H7,5
Error in line 52: a3c A822 29,7
Error in line 56: gkiwUA3u A20 50,9
Error in line 57: a3c S403 3,5
Error in line 64: S565 S565 55,8
Error in line 72: k3o A20 92,1
Error in line 95: S565 A20 84,2
Error in line 96: S565 S565 77,7
Error in line 94: a3c A20 1,3
Error in line 99: k3o S565 40,7
Error in line 102: a3c A822 10,8
Error in line 46: W7n9 S565 32,6
Error in line 107: W7n9 A20 7,9
Error in line 116: gkiwUA3u A20 20,1
Error in line 115: W7n9 A822 25,1
Error in line 124: S565 A20 68,8
Error in line 126: S565 S565 90,9
Error in line 133: gkiwUA3u A20 64,4
Error in line 143: W7n9 S403 10,5
Error in line 146: a3c A822 277,5
Error in line 155: a3c S565 79,2
Error in line 156: a3c A20 60,7
Error in line 165: k3o A20 134,82
Error in line 154: PqM S403 51,9
Error in line 167: PqM A822 53,4
Error in line 153: gkiwUA3u S565 72,8
Error in line 163: k3o A20 69,2
Error in line 182: a3c S403 92,8
Error in line 178: gkiwUA3u A822 30,8
Error in line 188: W7n9 A822 52,5
Error in line 192: a3c S565 49,7
Error in line 189: PqM A20 96,0
Error in line 208: oga S565 ;5,2
Error in line 209: S565 S565 148,14
Error in line 215: PqM A20 55,0
Error in line 217: W7n9 S565 3,0
Error in line 221: oga S565 58,2
Error in line 235: gkiwUA3u S565 0,1
Error in line 204: S565 S565 78,1
Error in line 237: gkiwUA3u A20 42,3
Error in line 242: S565 A20 -1,0
