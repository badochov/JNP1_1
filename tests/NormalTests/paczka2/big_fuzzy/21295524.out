S565 A 10,6
W7n9 A 1,2
a3c S 17,8
gkiwUA3u A 14,9 S 7,2
oga A 120,7 S 0,2
A20 1,2
S403 18,0
S565 7,2
A822 146,2
A20 22,3
A20 22,3
S565 A 10,6
W7n9 A 20,2 S 72,6
a3c A 4,5 S 17,8
gkiwUA3u A 14,9 S 34,2
oga A 122,8 S 0,2
A20 22,3
S403 90,6
S565 34,2
A822 150,7
S565 A 10,6
S565 249,6
S565 A 10,6
S565 445,5
A822 243,9
PqM A 124,3 S 197,2
S565 A 192,8 S 7,2
W7n9 A 20,2 S 72,6
a3c A 46,8 S 35,2
gkiwUA3u A 14,9 S 145,0
k3o A 98,8 S 316,3
oga A 213,8 S 26,6
A20 436,6
S403 321,0
S565 479,1
A822 275,0
S403 331,5
A20 481,7
oga A 213,8 S 26,6
PqM A 124,3 S 201,9
S565 A 202,2 S 7,2
W7n9 A 20,2 S 72,6
a3c A 82,5 S 35,2
gkiwUA3u A 14,9 S 155,5
k3o A 98,8 S 316,3
oga A 213,8 S 26,6
A20 481,7
S403 336,2
S565 479,1
A822 275,0
S565 A 202,2 S 7,2
S565 479,1
PqM A 124,3 S 201,9
S403 409,9
A822 904,0
PqM A 124,3 S 248,3
S565 A 218,6 S 7,2
W7n9 A 287,2 S 300,3
a3c A 102,0 S 108,9
gkiwUA3u A 287,5 S 410,4
k3o A 98,8 S 838,9
oga A 267,3 S 26,6
A20 481,7
S403 796,1
S565 1144,5
A822 904,0
k3o A 236,1 S 838,9
oga A 449,8 S 26,6
A20 594,7
PqM A 131,0 S 248,3
S565 A 291,0 S 221,5
W7n9 A 652,9 S 300,3
a3c A 132,4 S 108,9
gkiwUA3u A 287,5 S 410,4
k3o A 236,1 S 838,9
oga A 449,8 S 26,6
A20 594,7
S403 1010,4
S565 1144,5
A822 1586,0
A822 1606,8
PqM A 186,3 S 302,9
oga A 449,8 S 98,1
PqM A 186,3 S 391,9
S565 A 300,7 S 221,5
W7n9 A 859,6 S 366,8
a3c A 328,8 S 127,5
gkiwUA3u A 556,1 S 729,8
k3o A 353,0 S 908,3
oga A 449,8 S 234,1
A20 1132,8
S403 1682,7
S565 1297,2
A822 1901,5
PqM A 186,3 S 391,9
S565 A 300,7 S 221,5
W7n9 A 859,6 S 366,8
a3c A 328,8 S 127,5
gkiwUA3u A 556,1 S 729,8
k3o A 353,0 S 908,3
oga A 449,8 S 234,1
A20 1132,8
S403 1682,7
S565 1297,2
A822 1901,5
